core/src/main/java/org/apache/accumulo/core/constraints/Violations.java

```diff
@@ -80,17 +56,7 @@ private Violations(Map<CVSKey,ConstraintViolationSummary> cvsmap) {
    * @return true if empty
    */
   public boolean isEmpty() {
-    return cvsmap.isEmpty();
-  }
-
-  private void add(CVSKey cvsk, ConstraintViolationSummary cvs) {
-    ConstraintViolationSummary existingCvs = cvsmap.get(cvsk);
-
-    if (existingCvs == null) {
-      cvsmap.put(cvsk, cvs);
-    } else {
-      existingCvs.numberOfViolatingMutations += cvs.numberOfViolatingMutations;
-    }
+    return super.isEmpty();
```

Comment:
Could just delete these methods that are now proxying the super class implementation, so it inherits the super implementation directly, unless you need this to override the javadoc.

Reply:
I think I am going to just copy this class over too with the others. We won't need any changes then.

Reply:
I think this class was already copied over. Keith mentioned above that this one might be only needed internally, and not necessary to move, though.

My point was different. My point was a generic Java observation about subclasses not needing to have an explicit override if merely calls `super.whatever()`. If you just delete the method in the subclass, it will automatically inherit from the superclass implementation, and it will make the subclass smaller and easier to read/maintain.

Reply:
Sorry I was thinking the of a different class, disregard that comment.